src/applications/personalization/profile360/vet360/containers/InitializeID.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import AlertBox from '@department-of-veterans-affairs/formation/AlertBox';
+
+import {
+  INIT_VET360_ID,
+  ANALYTICS_FIELD_MAP,
+  API_ROUTES,
+  VET360_INITIALIZATION_STATUS
+} from '../constants';
+
+import {
+  createTransaction,
+  fetchTransactions,
+  refreshTransaction
+} from '../actions';
+
+import {
+  selectVet360InitializationStatus
+} from '../selectors';
+
+import TransactionPending from '../components/base/TransactionPending';
+
+class InitializeVet360ID extends React.Component {
+  componentDidMount() {
+    if (this.props.status === VET360_INITIALIZATION_STATUS.INITIALIZED) {
+      this.props.fetchTransactions();
+    } else {
+      this.initializeVet360ID();
+    }
+  }
+
```

Comment:
At mount, the status has to be either `UNINITIALIZED` or `INITIALIZED` (not pending or anything else), so we either fetch the transactions or do the initialization based on that.